### PR TITLE
Fixes #31 - Update hero section on homepage to match new design

### LINKED ISSUE #31: UXD: Add new hero HTML and CSS

### PR DESCRIPTION
**Pull request by ChrisGoodwin**

### CODE REVIEW

File: app/styles/layout.css [hunk 1]
```diff
@@ -22,6 +12,31 @@
...
+  padding-right: 10%;
+  padding-left: 10%;
+  background:
+    repeating-linear-gradient(
+       45deg,
+       #fdf8ee,
+       #fdf8ee 6px,
+       #fbf4e5 6px,
+       #fbf4e5 8px
+    );
```
Comment (Ctannerweb) [inline app/styles/layout.css]: background can be on one line and colors should be variables.

File: app/styles/layout.css [hunk 2]
```diff
@@ -22,6 +12,31 @@
     margin-bottom: 100px;
   }
 }
+
+.l-header {
+  box-sizing: border-box;
```
Comment (Ctannerweb) [inline app/styles/layout.css]: Not needed

File: app/styles/modules/header.css [hunk 3]
```diff
@@ -14,7 +13,6 @@
   color: var(--gray);
```
Comment (Ctannerweb) [inline app/styles/modules/header.css]: .header__headings div is not needed. Move `text-align` to `l-header`

**Comment by hbrysiewicz** [comment]
image

Looks like this header change affected other pages as whale

**Comment by Ctannerweb** [comment]
@hbrysiewicz this branch should not be merged into master. This is our "re-design master" branch.

**Pull request merged**